KakapoExample/KakapoTests/checkUrlTest.swift

```diff
@@ -15,24 +15,26 @@ class checkUrlTest: QuickSpec {
     override func spec() {
         describe("#checkUrl") {
             it("should return nil if the requested url doesn't match the declared one") {
-                expect(parseUrl("/users/:id", requestURL: "/comments/1")).to(beNil())
```

Comment:
And I guess decompose could need more tests (will skip the review of this tests for now)